cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -78,24 +80,20 @@ public Outlet retrieveOutlet(String officeId, String locationId) {
         });
     }
 
-    private CwmsId getRatingGroupId(String locationId, List<LocationGroup> groups) {
+    private Optional<LocationGroup> getRatingGroup(String locationId, List<LocationGroup> groups) {
         return groups.stream()
                      .filter(group -> group.getAssignedLocations()
                                            .stream()
                                            .anyMatch(loc -> loc.getLocationId().equalsIgnoreCase(locationId)))
```

Comment:
nitpick:
for readability, try to keep the stream as tight as possible
```
group.getAssignedLocations()
              .stream()
              .map(AssignedLocation::getLocationId)
              .anyMatch(id -> id.equalsIgnoreCase(locationId))
```

Reply:
This is my code, and he's right about this, I should have written it this way.

Reply:
Cleaned up stream